ckanext/bpatheme/templates/snippets/bpa_register.html

```diff
@@ -2,7 +2,7 @@
 {% if not c.userobj %}
 <div class="row">
 <div class="span2"></div>
-<div class="span5" style="font-size: 130%; text-align: center;">
+<div class="span" style="font-size: 130%; text-align: center;">
```

Comment:
is this meant to be in here? it looks like a mistake?

Reply:
Its just a small change so I thought to push to all together.

Reply:
@grahame  is this ok?

Reply:
I'm pretty sure 'span' isn't a defined CSS class; 'span5' is a bootstrap 2 class that makes an element span 5 columns... why are you making this change?